cmd/loom/loom.go

```diff
@@ -833,7 +834,14 @@ func loadApp(
 					return nil, err
 				}
 			}
-			return evm.NewLoomVm(state, eventHandler, receiptHandlerProvider.Writer(), createABM, cfg.EVMDebugEnabled), nil
+			return evm.NewLoomVm(
+				state,
+				eventHandler,
+				receiptHandlerProvider.Writer(),
+				createABM,
+				loomprecompiles.NewLoomPrecompileHandler(getContractStaticCtx("addressmapper", vmManager)),
```

Comment:
The new precompiles should be behind a feature flag, so need to call `state.FeatureEnabled` here, if it's not enabled don't pass through the precompiles.